api/src/public/webapp/manifest.appcache

```diff
@@ -1,4 +0,0 @@
-CACHE MANIFEST
-
-NETWORK:
-*
```

Comment:
I think we still need this for anyone who has yet to migrate to service workers.

Reply:
Wait ... this should still be there, just moved.

Reply:
I merged an updated branch and I think things got messed up, but I had thought I fixed this. Definitely was intending for this to exist!

Reply:
Added it back, it was a merge error.